src/pages/layer-2.js

```diff
@@ -199,21 +221,27 @@ const Layer2Page = ({ data }) => {
     const fetchCryptoStats = async () => {
       try {
         // Average eth transfer fee from L2's supported by cryptostats API
-        const feeData = await getData(
+        let feeData = await getData(
           "https://api.cryptostats.community/api/v1/l2-fees/feeTransferEth?metadata=false"
         )
+
+        // Temporary, but filtering out L2's we arent listing
+        feeData = feeData.data.filter(
+          (l2) => l2.id !== "aztec-network" && l2.id !== "hermez"
+        )
+
         const feeAverage =
-          feeData.data.reduce(
+          feeData.reduce(
             (acc, curr) => (acc += curr.results.feeTransferEth),
             0
-          ) / feeData.data.length
+          ) / feeData.length
 
         const intlFeeAverage = new Intl.NumberFormat(intl.locale, {
           style: "currency",
           currency: "USD",
           notation: "compact",
           minimumSignificantDigits: 2,
-          maximumSignificantDigits: 3,
+          maximumSignificantDigits: 2,
```

Comment:
Currently, the 'Average layer 2 ETH transfer fee' is showing up as 1 decimal point. I was verrrry confused by this because in school, we were taught decimal places ==== significant digits. [Apparently not!](https://simple.wikipedia.org/wiki/Arithmetic_precision)

<img width="558" alt="Screenshot 2022-04-01 at 21 20 32" src="https://user-images.githubusercontent.com/62268199/161339779-623e52be-f97d-4408-80a4-c0e6d5f9cf76.png">

Not 100% sure if this is what you're after but it looks like there is a [`minimumFractionDigits` and `maximumFractionDigits`](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Intl/NumberFormat/NumberFormat) that we could use here to get it to display like regular currency ($1.10, in this case).

Tested a little and it seems to be working as we would expect.

![Screenshot 2022-04-01 at 21 49 19](https://user-images.githubusercontent.com/62268199/161339957-7c8f2dd8-edab-44f6-9c83-8b9d32f5f9ae.png)
![Screenshot 2022-04-01 at 21 49 39](https://user-images.githubusercontent.com/62268199/161339960-209cd54f-822e-44b2-bb5f-d5a758c96108.png)